Article/Article.js

```diff
@@ -3,24 +3,26 @@
 class Article {
   constructor(element) {
     // assign this.element to the passed in article element
-    this.element;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.element = element;
+    // create a reference to the ".expandButton" class. see line 10 and 11, here i chained the .textContent method instead
```

Comment:
Is the return value into `this.expandButton` the same? If so, that's cool, otherwise you want to break them out so that you retain the ref. Let me know, I'm curious.